src/Templates/ProductSearchResultsTemplate.php

```diff
@@ -8,10 +8,6 @@
  */
 class ProductSearchResultsTemplate {
 
-	const SLUG        = 'product-search-results';
```

Comment:
I think that if we will keep these constants and using them the code will look more clear. What do you think?

Reply:
Hmm, I believe we should just follow the pattern here: https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/f87c8ee48121a02a5b32e693de696509ec22040d/src/BlockTypes/ClassicTemplate.php#L64 - since all of the template slugs are hardcoded, I believe we should do the same with the search template

Reply:
Our code adds this template, and it is better to isolate the logic/names in the template to improve the readability of the code.